apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.  Includes the TutorialExplorer React class.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Immutable from 'immutable';
+import update from 'react-addons-update';
```

Comment:
@breville This is not in `package.json` but required here, so it's causing build [errors](https://cdo-build-logs.s3.amazonaws.com/staging/20160915T171903%2B0000?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJKADNJLDLSFUAK2Q%2F20160915%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20160915T172057Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=bd01081520558c3f7a41e8127cd67668c5ab7124bb574ed80809b222e56119dc). Reverting so you can fix at your leisure...